lib/searchkick.rb

```diff
@@ -59,6 +59,24 @@ def self.client
     end
   end
 
+  def self.client_v3
+    client_url = ENV["ELASTICSEARCH_V3_URL"]
+    return nil if client_url.blank?
+
+    @client_v3 ||=
+      Elasticsearch::Client.new({
+        url: client_url,
```

Comment:
This logic can be extracted and shared between `def self.client_v3` and `def self.client`

Reply:
totally agree, but the thing is that we can't debug this code and its pretty risky to do a refactor here, so for now i left this as-is